libp2p/crypto/crypto.nim

```diff
@@ -423,6 +424,40 @@ proc `$`*(key: PublicKey): string =
     result.add($(key.skkey))
     result.add(")")
 
+func shortLog*(key: PrivateKey): string =
+  ## Get string representation of private key ``key``.
+  if key.scheme == RSA:
+    result = ($key.rsakey).shortLog
+  elif key.scheme == Ed25519:
+    result = "Ed25519 key ("
+    result.add(($key.edkey).shortLog)
```

Comment:
I think this PR should be merged sooner rather than later, so don't let this delay you, but don't you find `shortLog($key.edkey)` easier to read than `($key.edkey).shortLog`?

Reply:
Good point, 
It seems that I've grown this habit due to nim syntax freedom...
It seems that I do this when there is a unary type/cast/transform.
I'm down for killing the habit, does not matter really much 😄 
But in that case I would regex the whole PR!

Reply:
lol, if we're going into that kind of detail here, ` key(` is redundant for logging and a simple `:` would do (`ed25519:12345abcdef`) - ie it's usually known from context that it's a key. if we wanna go nuts here and save space, we could also use the multicodec prefix: https://github.com/multiformats/multicodec/blob/master/table.csv - using the multihash format: https://multiformats.io/multihash/#the-multihash-format